### PR TITLE
DM Registration Page

### PR DESCRIPTION
**Pull request by Benjamin-Frost**
Closes: #13

### CODE REVIEW

**Review by JonasWanke — CHANGES_REQUESTED** [review]

File: src/forms/dm-registration-form.tsx [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+        <div className="flex justify-end">
+          <Button type="submit" isLoading={isSubmitting}>
+            Absenden
+          </Button>
+        </div>
+      </div>
+    </form>
+  );
+}
```
Comment (CodesDoWork) [inline src/forms/dm-registration-form.tsx:264]: TODO: add localization, but ok for now

**Review by CodesDoWork — CHANGES_REQUESTED** [review]

**Review by JonasWanke — APPROVED** [review]
(I haven't looked at the code in detail)

**Review by CodesDoWork — APPROVED** [review]

**Comment by CodesDoWork** [comment]
I think it would be great to have more details here (e.g. max age) before the actual registration formula.

**Comment by Benjamin-Frost** [comment]
> I think it would be great to have more details here (e.g. max age) before the actual registration formula.

Good suggestion! I added a small info box

**Pull request merged**